db/seeds/utils.js

```diff
@@ -20,3 +21,19 @@ exports.formatComments = (comments, idLookup) => {
     };
   });
 };
+
+exports.checkUserExists = (id) => {
+  return db
+    .query("SELECT username FROM users WHERE username=$1", [id])
+    .then(({ rows: username }) => {
+      return username;
+    });
+};
+
+exports.checkIDExists = (id) => {
+  return db
+    .query("SELECT article_id FROM articles WHERE article_id=$1", [id])
+    .then(({ rows: iD }) => {
+      return iD;
+    });
+};
```

Comment:
`checkUserExists` and `checkIDExists` are wildly similar. I would look into making one utility that can be used for both. The notes tell you how to do this!